bin/ensure-node-modules-present.js

```diff
@@ -0,0 +1,13 @@
+const spawnSync = require( 'child_process' ).spawnSync;
+const fs = require( 'fs' );
+
+if ( ! fs.existsSync( 'node_modules' ) ) {
```

Comment:
Oh, another thing is: I may have missed some conversations, but I don't understand why the install check is done differently in this hooks case.